parts/chapters/subsections/12.3/LGRLOCK.fodt

```diff
@@ -4265,7 +4265,7 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
     </table:table>
     <text:p text:style-name="P14570">Table <text:sequence text:ref-name="refTable591" text:name="Table" text:formula="ooow:Table+1" style:num-format="1">12.49</text:sequence>: LGRLOCK Keyword Description</text:p>
     <text:h text:style-name="P18589" text:outline-level="4"><text:bookmark-start text:name="__RefHeading___Toc716639_3964674244"/>Example<text:bookmark-end text:name="__RefHeading___Toc716639_3964674244"/></text:h>
-    <text:p text:style-name="_40_TextBody">The example below defines three oil LGRs(LGR-OP01,-OP02, and -OP03) and all the gas well LGRs (LGR-GP*) that should have their independent solution time steps deactivated.</text:p>
+    <text:p text:style-name="_40_TextBody">The example below defines three oil LGRs(<text:a xlink:href="#__RefHeading___Toc55049_4106839650">LGR</text:a>-OP01,-OP02, and -OP03) and all the gas well LGRs (<text:a xlink:href="#__RefHeading___Toc55049_4106839650">LGR</text:a>-GP*) that should have their independent solution time steps deactivated.</text:p>
```

Comment:
I think it would be best not to create links where the KEYWORD is pre-fixed or post-fixed by other characters (for example "LGR-OP01", "LGR-GP*"). Perhaps we could catch these in the script?

Reply:
Yes, but we need to accept keywords like `KRNUMX-`. That is: keywords can end with a dash.

Reply:
Updated the script to address this, see: https://github.com/OPM/opm-reference-manual/pull/439